src/main/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetSSLContextRpcClientBuilder.scala

```diff
@@ -0,0 +1,24 @@
+package io.iohk.ethereum.faucet.jsonrpc
+
+import com.typesafe.config.ConfigFactory
+import io.iohk.ethereum.security.{SSLConfig, SSLContextFactory, SSLError, SecureRandomBuilder}
+import javax.net.ssl.SSLContext
+
+trait FaucetSSLContextRpcClientBuilder {
+  self: SecureRandomBuilder =>
+
+  private lazy val sslConfig: Option[SSLConfig] = SSLConfig(ConfigFactory.load().getConfig("faucet"))
+
+  lazy val sslContextRPCClient: Either[SSLError, SSLContext] =
```

Comment:
I think it could be abstracted (is the same in `SSLContextBuilder`)

Reply:
Done